blueprints/app/files/.eslintrc.js

```diff
@@ -2,12 +2,39 @@ module.exports = {
   root: true,
   parserOptions: {
     ecmaVersion: 2017,
-    sourceType: 'module'
+    sourceType: 'module',
   },
   extends: 'eslint:recommended',
   env: {
-    browser: true
+    browser: true,
   },
   rules: {
-  }
+  },
+  overrides: [
+    // node files
+    {
+      files: [
+        './ember-cli-build.js',
+        './testem.js',
+        './index.js',
+        'config/**/*.js',
+      ],
+      parserOptions: {
+        sourceType: 'script',
+        ecmaVersion: 2015,
+      },
+      env: {
+        browser: false,
```

Comment:
I think this has to be `browser: null` or something like that. iirc `true/false` only defines whether the global can be overwritten or something like that

Reply:
Really!?!?! Weird! I’ll look it up. I have used this on a few other projects, will test and update as needed.

Reply:
I just realized I might be confusing this with the stuff in the `globals` object, but we should verify just in case.

Reply:
Ya, I was looking through the docs on configuring and I think you are thinking of the globals stuff.  For easy cross referencing:

* https://eslint.org/docs/user-guide/configuring#specifying-environments <- Docs on these `env` flags (no discussion of clobbering)
* https://eslint.org/docs/user-guide/configuring#specifying-globals <- Docs on `globals` (includes discussion on `true` / `false` flag to allow clobbering)